tests/integration/telemetry/stats/prometheus/stats.go

```diff
@@ -278,6 +324,33 @@ func buildQuery(sourceCluster string) (sourceQuery, destinationQuery, appQuery s
 	return BuildQueryCommon(labels, ns.Name())
 }
 
+func buildOutOfMeshServerQuery(sourceCluster string) string {
+	ns := GetAppNamespace()
+	labels := map[string]string{
+		"request_protocol":               "http",
+		"response_code":                  "200",
+		"destination_app":                "unknown",
```

Comment:
comments: additional comment is useful here as to why destination_service is set but not app and version.